src/it/resources/com/google/checkstyle/test/chapter2filebasic/rule233nonascii/InputAvoidEscapedUnicodeCharacters.java

```diff
@@ -17,15 +17,15 @@ public Object fooString()
         String fakeUnicode = "asd\tsasd"; 
         String fakeUnicode2 = "\\u23\\u123i\\u";
         String content = "";
-        return "\ufeff" + content ; // byte order mark ok
+        /*byte order mark ok*/return "\ufeff" + content ;
```

Comment:
According to Google Style, characters with a tail comment and non-printable ones are both valid. We cannot determine which configuration plays a role using the original codes. Assuming that Google no longer allows the use of non-printable characters one day in the future (I know the possibility is very low), the test will still pass for the tail comments. In that case, we are missing the validation of non-printable characters in integration test. In other word, if our check of non-printable characters raise a false negative, we are not able to find it via this test.